integration_tests/integration/checkAnswers.spec.ts

```diff
@@ -50,6 +56,10 @@ context('Check Answers', () => {
     const confirmArrivalPage = ConfirmArrivalPage.goTo(expectedArrival.id)
     confirmArrivalPage.continue().click()
 
+    const sexPage = SexPage.goTo(expectedArrival.id)
```

Comment:
Again here - don't need to goTo - just verify that you've landed on the correct page

Reply:
Add a `checkAnswersPage.sex().should('contain.text', 'Female')`